sycl/test-e2e/ESIMD/unified_memory_api/gather_usm_dg2_pvc.cpp

```diff
@@ -26,16 +26,16 @@ int main() {
   constexpr auto TestFeatures = TestFeatures::DG2;
   bool Passed = true;
 
-  Passed &= testUSM<int8_t, TestFeatures>(Q);
-  Passed &= testUSM<int16_t, TestFeatures>(Q);
+  Passed &= testSLM<int8_t, TestFeatures>(Q);
```

Comment:
Are these changes correct? I would expect we want to call testUSM in the usm*.cpp file.